dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/client/ServiceDiscoveryRegistryDirectory.java

```diff
@@ -339,9 +339,7 @@ private void refreshInvoker(List<URL> invokerUrls) {
             "Registry type: instance. " +
             "Service Key: " + getConsumerUrl().getServiceKey() + ". " +
             "Urls Size : " + invokerUrls.size() + ". " +
-            "Invokers Size : " + getInvokers().size() + ". " +
-            "Available Size: " + getValidInvokers().size() + ". " +
-            "Available Invokers : " + joinValidInvokerAddresses());
```

Comment:
Why remove these lines?

Reply:
I will fix it.

Reply:
Recovered, For example:
![image](https://github.com/apache/dubbo/assets/18413695/d17caaf8-152d-4448-a5d8-55329e9a6dd5)